cmake_templates/zeek-config.in

```diff
@@ -18,6 +21,19 @@ version="@ZEEK_VERSION_FULL@"
 zeek_dist="@ZEEK_DIST@"
 zeekpath="@DEFAULT_ZEEKPATH@"
 
+report_feature() {
+    # $1: test result
```

Comment:
`$1` should be the CMake variable no the result,

```suggestion
    # $1: CMake feature flag
```

Reply:
It's the value of `have_XXX`, so a test result. I've reworded anyway.